pkgs/misc/vscode-extensions/wakatime/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchgit, vscode-utils }:
+
+let
+  inherit (vscode-utils) buildVscodeMarketplaceExtension;
+
+  cliVer = "10.2.1";
+  commandLineInterface = stdenv.mkDerivation {
```

Comment:
We have already a wakatime package, can you reuse it here?

Reply:
It seems to directly call `cli.py` from the it's own internal download. The same files seem to located in `"${wakatime}/lib/python3.6/site-packages/wakatime"` so I'll try that. Should I copy over the `__pycache__` as well?
```
  ERR EROFS: read-only file system, open '/nix/store/wgic0b48qr7s0gl1540j6ivqp5z92i06-vscode-extension-WakaTime-vscode-wakatime-1.2.2/share/vscode/extensions/WakaTime.vscode-wakatime/out/wakatime-master.zip': Error: EROFS: read-only file system, open '/nix/store/wgic0b48qr7s0gl1540j6ivqp5z92i06-vscode-extension-WakaTime-vscode-wakatime-1.2.2/share/vscode/extensions/WakaTime.vscode-wakatime/out/wakatime-master.zip'
```
```
/nix/store/rsa1rsd6c6szy0dmi61599b33aqiy269-python3-3.6.5/bin/python3: can't open file '/nix/store/wgic0b48qr7s0gl1540j6ivqp5z92i06-vscode-extension-WakaTime-vscode-wakatime-1.2.2/share/vscode/extensions/WakaTime.vscode-wakatime/out/wakatime-master/wakatime/cli.py': [Errno 2] No such file or directory
```

Reply:
Works with current wakatime version though it still attempts to update. Should be fixed with #42238.